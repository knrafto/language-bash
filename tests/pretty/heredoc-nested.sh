for x in true; do cat <<EOF; done
here doc
EOF

for x in true; do
    for y in true; do
        cat <<EOF
here doc
EOF
        true
    done
done

while true; do cat <<EOF; done
here doc
EOF

while true; do
    while true; do
        cat <<EOF
here doc
EOF
        true
    done
done

while true <<EOF
here doc
EOF
do
    true
done
