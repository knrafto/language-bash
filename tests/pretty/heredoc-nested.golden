for x in true; do
    cat <<EOF;
here doc
EOF
done;
for x in true; do
    for y in true; do
        cat <<EOF;
here doc
EOF
        true;
    done;
done;
while true; do
    cat <<EOF;
here doc
EOF
done;
while true; do
    while true; do
        cat <<EOF;
here doc
EOF
        true;
    done;
done;
